tidy3d/components/medium.py

```diff
@@ -3843,6 +4167,9 @@ def eps_dataarray_freq(
             for ind, mat_component in enumerate(self.components.values())
         )
 
+    def _sel_custom_data_inside(self, bounds: Bound):
+        return self
+
```

Comment:
The reduction doesn't seem to apply to anistoropic custom medium?

Reply:
It does apply. It's just a proper `sel_inside()` is already inherited from `AnisotropicMedium` and we just need to neutralize abstract `_sel_custom_data_inside` inherited from `AbstractCustomMedium`